bigquery_etl/copy_deduplicate.py

```diff
@@ -190,15 +157,21 @@ def _get_query_job_configs(
         ddl += f"\nPARTITION BY\n  DATE({stable_table.time_partitioning.field})"
         if stable_table.clustering_fields:
             ddl += f"\nCLUSTER BY\n  {', '.join(stable_table.clustering_fields)}"
-        ddl += "\nOPTIONS\n  (expiration_timestamp = "
-        ddl += "TIMESTAMP_ADD(CURRENT_TIMESTAMP, INTERVAL 1 DAY))"
+        ddl += (
+            "\nOPTIONS"
+            "\n  ("
+            "\n    partition_expiration_days = CAST('inf' AS FLOAT64),"
+            "\n    expiration_timestamp = "
+            "TIMESTAMP_ADD(CURRENT_TIMESTAMP, INTERVAL 1 DAY)"
+            "\n  )"
+        )
```

Comment:
You've verified that `partition_expiration_days = CAST('inf' AS FLOAT64)` works to disable partition expiration even in an anonymous dataset? Is this documented anywhere as a way to disable expiration?

Reply:
> You've verified that `partition_expiration_days = CAST('inf' AS FLOAT64)` works to disable partition expiration even in an anonymous dataset? Is this documented anywhere as a way to disable expiration?

It doesn't actually disable the expiration, but it does prevent partitions from expiring because it results in `timePartitioning.expirationMs = 9223372036828800000`, which is maxint rounded down to the nearest day, or ~292 million years.